### DISCUSSION TITLE
My idea to raise 15m

### DISCUSSION THREAD

**Discussion by Slum12** [iota-community/Shimmer-Governance#8] [Ideas]
Just sell a portion of the token supply. say 9/10 goes to iota holder, 1/10 is sold for 15 million. Iota has a marketcap at nearly 4 billion. Buying a 1/10th for 15 million make the marketcap of SMR at around 150m, so only a 1/26th of Iota. If more then 1/10th is sold for 15m i would even put my xmas money into shimmer, that is supposed for a vacation, because it would be that cheap :D

Since i think there is alot of potential in SMR, i could imagine the endprice beeing around 1/10th of iota. So it is an easy 3x. Hans said he thinks it might even be eventually higher then Iota, so it would be an even better investment for buyers.

And there is no weird stuff with increased rewards for staking. Just plain old buying a part of the total supply for a reasonable price to help start the network.

Why i think this is better: It values the 1000 Dollar of the small investor just as much as the bigger investments of the big guys. Everyone gets SMR for the same price. With additional staking rewards it clearly favors big bags.

(Posted this on reddit, but  here it might reach the right people)

Edit: And, as mentioned by someone else on reddit, you can still give out the NFTs, so people got bragging rights on beeing an early adopter.

**Comment by ACC30** [comment]
perhaps this is the difficulty of translation.  sorry but could you clarify for me.  the amount of shimmer coins will depend on the iota stacking into the shimmer network.  do you offer to take 10% from the stackers and put it on the stock exchange?  I am afraid that the formed price will become a huge slab below which the shimmer of the coin will be sold by stackers.

**Reply by Slum12** [reply]
My idea:
90% of SMR through staking

10% is sold more or less per ICO.
10% of SMR for 15m, that would value the network at 150m total, 1/26th of Iota.
I think the "real value" will probably be more like 1/10th of Iota, so early investors will profit.

Of course the 10% ICO sell is only a suggestion. I wanted to propose 20-25% at first, but after looking at the numbers i prefer 10%.
It could even be 50/50, to make absolutly sure all 15m are sold. But even at 20% i would cancel my xmas vacation to put the money into SMR :D

I think this approch is more fair, since people spending 9k for 3 gold NFTs all getting the same share of SMR, regardless of the amount of Iota they hold. Maybe someone wants to invest heavily in SMR, but isn't an Iota whale yet?

**Reply by 8bn-io** [reply]
@Slum12 Would be great if IF could postpone the Genesis NFT sale until after the lunch of the Shimmer network. Everyone who "staked" their $MIOTA in Firefly could participate and this would even increase the incentive to stake IOTA.

**Comment by sbruinsje** [comment]


**Reply by Slum12** [reply]
If you sell the NFTs like i proposed, then SMR would be valued at around 1/26th of Iota, which is probably way lower then it will be.
Everyone investing will probably tripple their investment shortly after it can be traded. 

But to be sure, you could also sell 20% or even 30%. 

Also you can still sell the NFTs, just give away a flat amount of SMR for each sell.

You could even have some kind of "bidding". Where you can first buy 10% of the network, if not enough people buy it then 15% a week later and so on, until everything is sold. At the end everyone who bought NFTs gets the same amount for their money. If the NFT was sold out at 10% then everyone gets a combinded 10%. If everything was sold out at 30, everyone gets combined 30%.

Then they will get their 15m 100%, and we will see how much of the network needs to be sold to get there.
And there is still no weird stuff going on, that might hinder the network in the longrun.

The Problem i see with your suggestion is, that at first probably only the big holders will buy themselfs out. Because they get alot of SMR for the same money little investors get their little bit. Its the same as with the IF proposal, a flat amount of money, for small or alot of gains depending on the stack of IOTA you hold. It will lock out small holders.
With my suggestion everyone gets the same amount of SMR for their money.
But I like your suggestion better then the one of the IF.
